sys/shell/commands/sc_gnrc_ipv6_nib.c

```diff
@@ -344,8 +344,8 @@ static int _nib_abr(int argc, char **argv)
             printf("address %s must be global\n", argv[3]);
             return 1;
         }
-        if (((netif = gnrc_netif_get_by_ipv6_addr(&addr)) == NULL) ||
-            gnrc_netif_is_6lbr(netif)) {
+        if (!(((netif = gnrc_netif_get_by_ipv6_addr(&addr)) != NULL) &&
+              gnrc_netif_is_6lbr(netif))) {
```

Comment:
I'm pretty sure that `!(!a && b)` can be simplified to `a || !b`.

It was easier to understand before:
 - `gnrc_netif_get_by_ipv6_addr(&addr) == NULL` -> error OR
 - `!gnrc_netif_is_6lbr(netif)` -> error

Reply:
> I'm pretty sure that `!(!a && b)` can be simplified to `a || !b`.

See comments after https://github.com/RIOT-OS/RIOT/pull/10951#discussion_r323467893. The formular actually now is `!(a && b)`, so with an `||` it would become a double negation `!a || !b`.

Reply:
Ok, neither is a double negation. But with each set of parenthesis you need need, things get harder to process mentally.